pkgs/development/tools/build-managers/dub/default.nix

```diff
@@ -1,11 +1,11 @@
-{stdenv, fetchurl, curl, dmd, gcc, unzip}:
+{stdenv, fetchurl, curl, dmd, gcc, unzip, git}:
```

Comment:
Le 2015-04-14 18:11, Charles Cooper a écrit :
>> @@ -1,11 +1,11 @@
>> -{stdenv, fetchurl, curl, dmd, gcc, unzip}:
>> +{stdenv, fetchurl, curl, dmd, gcc, unzip, git}:
> 
> dub build calls git

Therefore you should put `git` in the `buildInputs`.